docs/hub/datasets-adding.md

```diff
@@ -81,6 +81,7 @@ The Hub natively supports multiple file formats:
 - CSV (.csv, .tsv)
 - JSON Lines, JSON (.jsonl, .json)
 - Parquet (.parquet)
+- Arrow (.arrow)
```

Comment:
We support Arrow streaming format but not the (more common) Arrow IPC file format.

```suggestion
- Arrow (streaming format) (.arrow)
```